python/ngraph/ops.py

```diff
@@ -168,6 +170,190 @@ def mvn(data, axes, normalize_variance, eps, name=None):
     return MVN(data, AxisSet(axes), normalize_variance, eps)
 
 
+@nameable_op
+def quantize(data, scale, zero_point, new_type, axes, round_mode, name=None):
+    # type: (Node, Node, Node, NumericType, Set[int], int, str) -> Node
```

Comment:
The `round_mode` should not be marked as `int` type but rather the enum type you created.